examples/maxcut-qaoa.yaml

```diff
@@ -5,197 +5,204 @@ apiVersion: io.orquestra.workflow/1.0.0
+        memory: "1Gi"
+        disk: "10Gi"
+    inputs:
+      - number_of_parameters: 2
+        type: int
+      - min_value: 0
+        type: int
+      - max_value: 3.14159265359
+        type: float
+    outputs:
+      - name: params
+        type: parameters
 
-- name: optimize-variational-circuit
-  passed: [get-maxcut-hamiltonian, get-grid, generate-random-initial-ansatz-params, generate-fixed-ansatz-params]
-  config:
-    runtime:
-      language: python3
-      imports: [z-quantum-core, z-quantum-optimizers, qe-qulacs, z-quantum-qaoa]
-      parameters:
-        file: z-quantum-optimizers/steps/optimizers.py
-        function: optimize_variational_circuit
-    resources:
-      cpu: "1000m"
-      memory: "1Gi"
-      disk: "10Gi"
-  inputs:
-  - ansatz_specs:
-      module_name: zquantum.qaoa.ansatzes.farhi_ansatz
-      function_name: QAOAFarhiAnsatz
-      number_of_layers: 1
-    type: specs
-  - backend_specs:
-      module_name: qequlacs.simulator
-      function_name: QulacsSimulator
-    type: specs
-  - optimizer_specs:
-      module_name: zquantum.optimizers.grid_search
-      function_name: GridSearchOptimizer
-      options: {"keep_value_history": true}
-    type: specs
-  - cost_function_specs:
-      module_name: zquantum.core.cost_function
-      function_name: AnsatzBasedCostFunction
-      estimation_method_specs:
-        module_name: zquantum.core.estimation
-        function_name: calculate_exact_expectation_values
-    type: specs
-  - qubit_operator: ((get-maxcut-hamiltonian.hamiltonian))
-    type: qubit-operator
-  - initial_parameters: ((generate-random-initial-ansatz-params.params))
-    type: parameters
-  - fixed_parameters: ((generate-fixed-ansatz-params.params))
-    type: parameters
-  - parameter_grid: ((get-grid.parameter-grid))
-    type: grid
-  outputs:
-  - name: optimization-results
-    type: optimization-results
-  - name: optimized-parameters
-    type: parameters
+  - name: optimize-variational-circuit
+    passed:
+      [
+        get-maxcut-hamiltonian,
+        get-points-list,
+        generate-random-initial-ansatz-params,
+      ]
+    config:
+      runtime:
+        language: python3
+        imports:
+          [z-quantum-core, z-quantum-optimizers, qe-qulacs, z-quantum-qaoa]
+        parameters:
+          file: z-quantum-optimizers/steps/optimizers.py
+          function: optimize_variational_circuit
+      resources:
+        cpu: "1000m"
+        memory: "1Gi"
+        disk: "10Gi"
+    inputs:
+      - ansatz_specs:
+          module_name: zquantum.qaoa.ansatzes.farhi_ansatz
+          function_name: QAOAFarhiAnsatz
+          number_of_layers: 1
+        type: specs
+      - backend_specs:
+          module_name: qequlacs.simulator
+          function_name: QulacsSimulator
+        type: specs
+      - optimizer_specs:
+          module_name: zquantum.optimizers
+          function_name: SearchPointsOptimizer
+        type: specs
+      - keep_history: True
```

Comment:
Shouldn't this be part of `optimizer_specs`?

Reply:
No, because it is now an argument to `minimize` not `__init__`

Reply:
Oh indeed!